src/main/kotlin/com/openlattice/chronicle/services/download/DataDownloadService.kt

```diff
@@ -151,6 +155,34 @@ class DataDownloadService(private val storageResolver: StorageResolver) : DataDo
         return PostgresDownloadWrapper(iterable).withColumnAdvice(cols.map { it.name })
     }
 
+    private fun getQuestionnaireResponsesHelper(studyId: UUID, questionnaireId: UUID): Iterable<Map<String, Any>> {
+        val cols = listOf(
+            PostgresColumns.PARTICIPANT_ID,
+            PostgresColumns.COMPLETED_AT,
+            PostgresColumns.QUESTION_TITLE,
+            PostgresColumns.RESPONSES
+        )
+        val iterable = BasePostgresIterable<Map<String, Any>>(
+            PreparedStatementHolderSupplier(
+                hds = storageResolver.getPlatformStorage(),
+                SurveysService.GET_QUESTIONNAIRE_SUBMISSIONS_SQL,
+                32768
```

Comment:
What is this magic number? Please define as a constant.

Reply:
Same number used by the other functions in the download service. Not sure why that number though